libs/portals/my-pages/health/src/screens/Bloodtype/Bloodtype.tsx

```
@@ -0,0 +1,69 @@
+    <IntroWrapper
+      title={formatMessage(messages.bloodtype)}
+      intro={bloodType?.description ?? formatMessage(messages.bloodtypeDesc)}
+      serviceProviderSlug={LANDLAEKNIR_SLUG}
+      marginBottom={6}
+      buttonGroup={[
+        <LinkButton
+          to={bloodType?.link ?? formatMessage(messages.bloodtypeLink)}
+          text={formatMessage(messages.readAboutBloodtypes)}
+          variant="utility"
+          icon="open"
+        />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)